inc/unifaun_send.inc

```diff
@@ -332,6 +343,33 @@ class Unifaun {
 
     $uni_receiver->addAttribute('rcvid', utf8_encode($this->asiakasrow["tunnus"]));
 
+    if (isset($this->postirow["noutopisteen_tunnus"]) and $this->postirow["noutopisteen_tunnus"] != '') {
```

Comment:
@mikamyllynen !empty() ois kompatimpi ja ajaa tässä saman asian ku isset(xxx) and xxx != ""